bundles/core/org.eclipse.smarthome.core/src/main/java/org/eclipse/smarthome/core/auth/dto/AuthenticationDTO.java

```diff
@@ -0,0 +1,21 @@
+/**
+ * Copyright (c) 2014-2016 by the respective copyright holders.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.eclipse.smarthome.core.auth.dto;
```

Comment:
Do we really need DTOs at all? They are normally only for sending such objects through the REST API or as events, but I doubt we want to do that with any of the auth stuff. So unless required, I would opt for removing them.